scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
```

Comment:
Well if you were to clear the database, there wouldn't be any metadata at all in there as metadata are manually entered through the GUI. 
Not sure how it's possible to have animal releases without a corresponding surgery though. I tried to do that through the dev environment but it returned an error ('Animal release must have at least one tagging.'), so I suspect that it was a bug that got fixed but the underlying errors not cleaned up.